### ISSUE TITLE
Process Sentinel-1 data pre September 2016.

### ISSUE THREAD

**Issue by annesol26** [GEUS-Glaciology-and-Climate/Sentinel-1_Greenland_Ice_Velocity#4]

**Comment by annesol26** [comment]
The "[Greenland Ice Velocity from Sentinel-1 Edition 3]" dataset has been extended back in time to January 2016.

**Comment by mankoff** [comment]
Should this ticket be closed if this is now done?

**Comment by mankoff** [comment]
See 42 files  added in v6.0 at http://doi.org/10.22008/FK2/ZEGVXU

**Comment by annesol26** [comment]
No, I decided to just post a comment -the S1 archive extends further back, so I still have to figure out if I want to go back to 2015 as well.

-A

Fra: Ken Mankoff ***@***.***>
Sendt: 21. november 2022 15:47
Til: GEUS-Glaciology-and-Climate/Sentinel-1_Greenland_Ice_Velocity ***@***.***>
Cc: Anne Munck Solgaard ***@***.***>; Author ***@***.***>
Emne: Re: [GEUS-Glaciology-and-Climate/Sentinel-1_Greenland_Ice_Velocity] Process Sentinel-1 data pre September 2016. (#4)


Should this ticket be closed if this is now done?
